.env

```diff
@@ -1,3 +1,3 @@
-KNUU_TIMEOUT=60m # The timeout for the tests
+KNUU_TIMEOUT=120m   # The timeout for the tests
 KNUU_BUILDER=docker # The builder to use for building images. docker or kubernetes
```

Comment:
same for this one. and the default now is `kaniko` i.e. k8s based builder

Reply:
should this just be removed then?

Reply:
I would remove them because that's something to be handled by the user and they should care about their env. The only thing `knuu` does and I am not sure if it is useful anymore is that it loads any `.env` file it finds. @jrmanes knows better about it.

Reply:
sounds like this is outdate, either way we shouldn't be in the habit of committing .env imo.

Reply:
This `.env` file was committed as an example for the developers.

Reply:
It isn't clear that the `.env` was an example as it isn't referenced anywhere. 
Using an `.env` file is pretty common practice so I don't think we need an actual file as an example, just have the env var options clearly documented. 

for future examples, if we need/require a `.env` file because we have things that need to be defined and don't have defaults, we should create a `.env.example`.